platformio.ini

```diff
@@ -1351,6 +1351,26 @@ extra_scripts        = ${common.extra_scripts}
   buildroot/share/PlatformIO/scripts/stm32_bootloader.py
   buildroot/share/PlatformIO/scripts/mks_encrypt.py
 
+#
+# MKS Robin Pro V2
+#
+[env:mks_robin_pro2]
+platform      = ${common_stm32.platform}
+extends       = common_stm32
+build_flags   = ${common_stm32.build_flags} -DHAL_HCD_MODULE_ENABLED  -DUSBHOST -DARDUINO_BLACK_F407VE
```

Comment:
> `-DARDUINO_BLACK_F407VE`

Does this serve a purpose? This looks like something accidentally copied from `STM32F407VE_black`.

Reply:
I’m using it just to be able to use the generic variant. It’s ok for testing purposes of most of the board features.